karton/core/asyncio/backend/direct.py

```diff
@@ -11,24 +11,23 @@
 from redis.asyncio.client import Pipeline
 from redis.exceptions import AuthenticationError
 
-from karton.core import Config, Task
+from karton.core.asyncio.backend.base import KartonAsyncBackendProtocol
 from karton.core.asyncio.resource import LocalResource, RemoteResource
-from karton.core.backend import (
+from karton.core.backend import KartonBind, KartonMetrics, KartonServiceInfo
+from karton.core.backend.direct import (
     KARTON_BINDS_HSET,
     KARTON_TASK_NAMESPACE,
     KARTON_TASKS_QUEUE,
     KartonBackendBase,
-    KartonBind,
-    KartonMetrics,
-    KartonServiceInfo,
 )
+from karton.core.config import Config
 from karton.core.resource import LocalResource as SyncLocalResource
-from karton.core.task import TaskState
+from karton.core.task import Task, TaskState
 
 logger = logging.getLogger(__name__)
 
 
-class KartonAsyncBackend(KartonBackendBase):
+class KartonAsyncBackend(KartonBackendBase, KartonAsyncBackendProtocol):
```

Comment:
Why inherit from `KartonAsyncBackendProtocol`? The idea of duck typing/protocols is that you don't need to explicitly do this.

```suggestion
class KartonAsyncBackend(KartonBackendBase):
```

(disregard in case I don't understand something).